ts/features/itwallet/identification/hooks/useNoCieBottomSheet.tsx

```diff
@@ -0,0 +1,102 @@
+import { StyleSheet, View } from "react-native";
+import {
+  IOButton,
+  Pictogram,
+  VSpacer,
+  VStack
+} from "@pagopa/io-app-design-system";
+import I18n from "../../../../i18n";
+import { useIOBottomSheetModal } from "../../../../utils/hooks/bottomSheet";
+import IOMarkdown from "../../../../components/IOMarkdown";
+import { ItwEidIssuanceMachineContext } from "../../machine/provider";
+
+const noCieContentKeys = [
+  "features.itWallet.identification.l3.mode.bottomSheet.noCie.content.firstAttention",
+  "features.itWallet.identification.l3.mode.bottomSheet.noCie.content.secondAttention",
+  "features.itWallet.identification.l3.mode.bottomSheet.noCie.content.thirdAttention"
+] as const;
+
+/**
+ * Hook to manage the no CIE bottom sheet
+ */
+export const useNoCieBottomSheet = () => {
+  const machineRef = ItwEidIssuanceMachineContext.useActorRef();
+  const navigateToL2Identification = () => {
+    machineRef.send({ type: "go-to-l2-identification" });
+    noCieBottomSheet.dismiss();
+  };
+
+  const Footer = () => (
+    <VStack space={12}>
```

Comment:
According to Figma, 16 is the correct spacing for buttons.
```suggestion
    <VStack space={16}>
```